src/components/Toolbar.vue

```diff
@@ -18,7 +18,7 @@
             span(v-if="$vuetify.breakpoint.mdAndUp") Account
             v-icon(v-if="$vuetify.breakpoint.mdAndUp") mdi-menu-down
         v-list
-          v-list-tile(v-for="(item,i) in accountItems" :key="i")
+          v-list-tile(v-for="(item,i) in AccountItems" :key="i")
```

Comment:
Should start with lower case.

Reply:
Apologies I don't know why I changed it to uppercase.